ReactWindows/ReactNative.Shared/Bridge/ReactBridge.cs

```diff
@@ -108,11 +110,35 @@ public void Dispose()
 
         private void ProcessResponse(JToken response)
```

Comment:
>ProcessResponse [](start = 21, length = 15)

Can we relabel this private method to `ProcessResponseImmediate` or something like that?